uo_sys_framework/AOLimitModule/cfgFunctions/common/fn_AOLimitInit.sqf

```diff
@@ -6,177 +6,177 @@ UO_FW_EXEC_CHECK(CLIENT);
 //[_logic,_area,_selectedSides,_entryMode,_airsetting,_AOMode,_softAOtime,_softAOtimeAir] passed array
 
 [{(!(isNull player)) && {(CBA_missionTime > 1)}}, {
-	params ["_args", "_unit"];
-	_args params ["_logic","_area","_selectedSides","_entryMode","","_AOMode","",""];
-	private ["_startedInside","_run","_arrayname","_enteredZone","_outSide","_softAOMode"];
-	if ((toUpper _AOMode) isEqualto "SOFT") then {_softAOMode = true} else {_softAOMode = false;};
+    params ["_args", "_unit"];
+    _args params ["_logic","_area","_selectedSides","_entryMode","","_AOMode","",""];
+    private ["_startedInside","_run","_arrayname","_enteredZone","_outSide","_softAOMode"];
+    if ((toUpper _AOMode) isEqualto "SOFT") then {_softAOMode = true} else {_softAOMode = false;};
```

Comment:
this can be written as

`private _softAOMode = (toUpper _AOMode) isEqualto "SOFT"`